src/pages/KyberDAO/StakeKNC/StakeKNCComponent.tsx

```diff
@@ -374,14 +383,24 @@ export default function StakeKNCComponent() {
           setAttemptingTxn(false)
           setTxHash(tx)
           setDelegateAddress('')
+          refetchGasRefundInfo()
```

Comment:
seem duplicate
make a function
```
refresh = ()=>{
setAttemptingTxn(false)
           setAttemptingTxn(false)
          setDelegateAddress('')
          refetchGasRefundInfo()
}
```

Reply:
It's existing code. We should not touch what is working well =))